eZ/Bundle/EzPublishLegacyBundle/Tests/Routing/DefaultRouterTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package.
```

Comment:
Actually not (valid in my code as well). But we'd better do one cleanup PR for this.